CHANGELOG.md

```diff
@@ -1,36 +1,47 @@
 # CHANGELOG
 
+## v0.4.2 (2022-10-20)
+
+- Fix a bug where the error data of a bad HTTP request (4xx or 5xx) was not stored as expected in cassettes, causing
+  empty error streams on replay.
+    - Error data for a bad HTTP request is now stored as the "body" in the cassette just like a good HTTP request
+      would, rather than needlessly stored in a separate "error" key. This more closely matches the behavior of EasyVCR C#.
+    - This is a breaking change for previously-recorded "error" cassettes, which will no longer replay as expected and
+      will need to be re-recorded (although likely never worked as expected in the first place).
+
 ## v0.4.1 (2022-10-19)
 
 - Fix a bug where the error stream of a bad HTTP request (4xx or 5xx) was not properly recreated on replay.
 
 ## v0.4.0 (2022-10-04)
 
-- New feature: Set expiration time for interactions (how long since it was recorded should an interaction be considered valid)
-  - Can determine what to do if a matching interaction is considered invalid:
-    - Warn the user, but proceed with the interaction
-    - Throw an exception
-    - Automatically re-record (cannot be used in `Replay` mode)
-- New feature: Pass in a custom Logger instance to EasyVCR to funnel log messages into your own logging setup (fallback: logs to console)
+- New feature: Set expiration time for interactions (how long since it was recorded should an interaction be considered
```

Comment:
Let's leave the previous formatting alone for the CHANGELOG

Reply:
Bad merge, will fix.